README.md

```diff
@@ -35,11 +35,15 @@ Then you can proceed to the following steps:
 ```sh
 # 1. Install all dependencies:
 yarn
-# 2. Patch cluster to enable local development flow:
+# 2 Patch cluster to enable local development flow:
 yarn start:prepare
-# 3. Run server locally:
+# 3 Export Che certificate
+chectl cacert:export
+# 4 Set CHE_SELF_SIGNED_MOUNT_PATH with the copied path from the previous step
+export CHE_SELF_SIGNED_MOUNT_PATH="... copied path ..."
+# 5. Run server locally:
```

Comment:
This will be a bit more convenient:

```suggestion
# 3. Export Che certificate
chectl cacert:export --destination=$TMPDIR
# 4. Set CHE_SELF_SIGNED_MOUNT_PATH
export CHE_SELF_SIGNED_MOUNT_PATH=$TMPDIR
# 5. Run server locally:
```

Reply:
I have fixed it